app/controllers/reacts_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# react emotion
+class ReactsController < ApplicationController
+  before_action :logged_in_user, only: %i[create destroy]
+  before_action :set_micropost, only: %i[create destroy]
+
+  def create
+    return render json: { success: false, status: 'error' } if React.find_by(user_id: params[:react][:user_id], micropost_id: params[:react][:micropost_id])
```

Comment:
Put into new  function `def react_exists?`  instead of ``` React.find_by(user_id: params[:react][:user_id], micropost_id: params[:react][:micropost_id])```